### ISSUE TITLE
Order of repositories is reversed

### ISSUE THREAD

**Issue by madoar** [PhoenicisOrg/phoenicis#1785]
It seems like the priority order of the repositories is reversed, i.e. the last added repository has the highest priority. This shouldn't be the case.

Possibly linked to #1756

**Comment by Kreyren** [comment]
Agree with possibly linked to #1758 but i'm not sure how to confirm.. it seems to appear on random?

**Comment by madoar** [comment]
You misunderstand me^^ I literally mean that the repositories are prioritized in the wrong order, i.e. the least important one is regarded as the most important one.

**Comment by Kreyren** [comment]
i'm even more confused now o.o

**Comment by plata** [comment]
Unit test?

**Comment by madoar** [comment]
Yes a unit test would be best, but I'm not sure how easily we can create multiple dummy repositories for testing.

**Comment by qparis** [comment]
You do not need to create dummy repositories to unit test. A mock is good enough
